css/dashboard/helpers.less

```diff
@@ -24,6 +24,7 @@
   line-height: @row-height;
   padding: @row-padding;
   border: @cell-border;
+  flex-grow: 1;
```

Comment:
I moved this into the basic `cell` css...I think it is a more reasonable default than `0`, since in general the cells want to take up the whole container.